build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+make -f MakefileVC
+objdump -a -d Diablo.exe | cut -f 3- | sed -E s/0x[0-9a-f]+/0xaddress/g > /tmp/Diablo.asm
```

Comment:
That `sed` command will remove all hexadecimal numbers, which also includes stack offsets.
I'm not sure if this won't remove too much info out of the asm file.